src/Client/Web/Bit.Client.Web.BlazorUI/Components/Navs/BitNav.razor

```diff
@@ -49,12 +49,11 @@
 
         childTemplate = (navLink) =>
         (
-    @<li role="listitem" @key="navLink.Key">
+    @<li role="listitem" @key="navLink.Key" @onclick="(() => Toggle(navLink))">
```

Comment:
this is not the correct way to fix this issue, it is making another issue that is related to bubbling the click event to this parent element from all of the child elements.